framework/include/vectorpostprocessors/VectorPostprocessorData.h

```diff
@@ -90,16 +128,32 @@ class VectorPostprocessorData : public Restartable
   const std::vector<std::pair<std::string, VectorPostprocessorState>> &
   vectors(const std::string & vpp_name) const;
 
+private:
+  // Forwad declaration
+  struct VectorPostprocessorVectors;
+
+public:
+  /**
+   * Get the map to VectorPostprocessorVectors for a particular VPP
+   */
+  VectorPostprocessorVectors & vectorPostprocessorVectors(const std::string & vpp_name)
+  {
+    return _vpp_data[vpp_name];
```

Comment:
RHS map brackets. Please add a mooseAssert or some other check.

Reply:
Removed this method